src/templates/finops-hub/schema/schema_ea_normalized.json

```diff
@@ -0,0 +1,723 @@
+        },
+        "sink": {
+          "name": "PreviousInvoiceId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "pricingCurrency",
+          "type": "String"
+        },
+        "sink": {
+          "name": "PricingCurrency",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "PricingModel",
+          "type": "String"
+        },
+        "sink": {
+          "name": "PricingModel",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "PartNumber",
+          "type": "String"
+        },
+        "sink": {
+          "name": "ProductId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "ProductName",
+          "type": "String"
+        },
+        "sink": {
+          "name": "ProductName",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "ProductOrderId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "ProductOrderId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "OfferId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "ProductOrderName",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "provider",
+          "type": "String"
+        },
+        "sink": {
+          "name": "Provider",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "publisherId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "PublisherId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "PublisherName",
+          "type": "String"
+        },
+        "sink": {
+          "name": "PublisherName",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "PublisherType",
+          "type": "String"
+        },
+        "sink": {
+          "name": "PublisherType",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "Quantity",
+          "type": "String"
+        },
+        "sink": {
+          "name": "Quantity",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "resellerMpnId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "ResellerMpnId",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "resellerName",
+          "type": "String"
+        },
+        "sink": {
+          "name": "ResellerName",
+          "type": "String"
+        }
+      },
+      {
+        "source": {
+          "name": "ReservationId",
+          "type": "String"
+        },
+        "sink": {
+          "name": "ReservationId",
```

Comment:
Does BenefitId/Name cover reservations? If so, should we just get rid of this column? I'm fine either way. Just asking.

Reply:
We could.  The benefitId field is a bit different to reservationId as it also includes the reservation order which is probably a good thing anyway.